CONTRIBUTING.md

```diff
@@ -146,3 +146,21 @@ and triggers release and publishing based on the following rules:
 | Commit with type `feat`            | Minor release |
 | Commit with type `fix`             | Patch release |
 | Commit with type `perf`            | Patch release |
+
+## Link widgets with local version of components/sdk-component-adapter
+
+First, in your local components/sdk-component-adapter folder, you need to run `npm run build` to generate the "dist"
+folder which will be used by widgets.
+
+Then, in your local widgets folder, do the following steps (ex. for components):
+
+- in package.json change
+  `"@webex/components": "^1.79.0",` with
+  `"@webex/file:../components"` (your local components path)
+- delete widgets/node_modules/@webex/components folder
+- run `npm install`
+- run `npm run link ../components/node_modules/react` (this is required ONLY for components)
+- run `npm run start`
+
+Keep in mind that for every modification made in components/sdk-component-adapter, you need to run `npm run build` in
```

Comment:
you need to re-run `npm run build` in the dependent repo.

Reply:
done